### ISSUE TITLE
Force all themes to use lowercase names

### ISSUE THREAD

**Issue by chris-roerig** [flazz/vim-colorschemes#66]
I've noticed that the themes using an uppercased name are missing out on tab completion (unless uppercase letter is specified) or even just basic colorscheme name cycling. This would be an easy change but would force existing users to update their vimrc.

**Comment by flazz** [comment]
do you have any data on how many existing color schemes violate this?

**Comment by ieei** [comment]
Piggybacking on this issue,

I was trying out monokai today. To my surprise it didn't load:
`colorscheme monokai`
```
Error detected while processing $HOME/.vimrc:
line    6:
E185: Cannot find color scheme 'monokai'
Press ENTER or type command to continue
```
so I changed to `colorscheme Monokai`
```
Error detected while processing /usr/share/vim/vim74/syntax/synload.vim:
line   19:
E185: Cannot find color scheme 'monokai'
Press ENTER or type command to continue
```
after that I did `mv Monokai.vim monokai.vim`
which now works

using:
```
VIM - Vi IMproved 7.4 (2013 Aug 10, compiled Oct 21 2015 13:03:49)
Included patches: 1-712
Modified by pkg-vim-maintainers@lists.alioth.debian.org
Compiled by buildd@
```

so yeah, this seems to work well on case sensitive filesystems &lt;/sarcasm&gt;

**Comment by flazz** [comment]
My current opinion is that this repo is true to upstream. If there is an issue preventing a colorscheme from being used maybe we take out the offending theme until upstream is fixed.

I'm open to discussion, but it might take a while.

**Comment by coreoas** [comment]
This is causing many issues with macos since the HFS+ file system is case-insensitive:

`colors/darkBlue.vim` and `colors/darkblue.vim` is clearly not working well in there

**Comment by alsemyonov** [comment]
And there is the same issue with `colors/darkBlue.vim` and `colors/darkblue.vim` on newer APFS. 

Git always report dirty state of repo, and there is no way to checkout clean master branch.

**Comment by xbeta** [comment]
I am not sure why is this not merged? The fix is obvious and simple, and it is impacting all case-insensitive filesystem (e.g. default APFS, Windows)

**Comment by chrispaterson** [comment]
I second the vote for this!

**Comment by flazz** [comment]
sorry all for the late reply,

I was able to reproduce much of the complaint. I just did a fresh checkout on mojave and did `:colorscheme Mono<TAB>` and it works. dark blue didn't tab complete for me but did work when typing `:colorscheme darkBlue` and git does show `colors/darkBlue.vim` as modified.

So it seems there is an issue with git+apfs; an issue with vim's autocompletion. Is there any upstream knowledge around these?

Mandating all files are lower case will solve these specific issues, but will the root causes manifest outside of latin case?

**Comment by chris-roerig** [comment]
I agree an upstream fix would be better but I wonder how long that would take to trickle out.

**Comment by flazz** [comment]
Someone want to get a branch (or three) up (apologies if one exists and I dismissed it) with changes specific to these files.

Let's get the changes out upstream too. Considering upstream traction we can work towards getting these into master.

Thanks for all the concern and patience folks.

**Comment by chris-roerig** [comment]
@flazz see https://github.com/flazz/vim-colorschemes/pull/156
